src/common/snippets/src/lowered/loop_info.cpp

```diff
@@ -350,6 +350,16 @@ const std::vector<int64_t>& ExpandedLoopInfo::get_data_sizes() const {
     return m_data_sizes;
 }
 
+void ExpandedLoopInfo::update_ptr_increments(const std::vector<int64_t>& new_values) {
```

Comment:
When we discussed [this comment](https://github.com/openvinotoolkit/openvino/pull/21922#discussion_r1601375422) my idea was to employ move semantics. So the vectors should be passed by value, not by const reference. Then (as we discussed offline) compiler will be able to create more efficient code without redundant memory allocation.